### PR TITLE
Adds bracelets!

### PR DESCRIPTION
**Pull request by mistyLuminescence**
Adds bracelets! Three versions so far.

Regular: really just a placeholder, but sure. It's a bracelet!
Material: made from different materials, like rings!
Friendship: made from rainbows and very pretty! Dedicate it to your friends!

Available in the loadout, and can be worn as accessories. They're tiny - 3x1 pixels when worn as far as sprites go - but they're _sentimental_.

### CODE REVIEW

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm [hunk 1]
```diff
@@ -209,3 +209,27 @@
...
+	bracelettype["bracelet, platinum"] = /obj/item/clothing/accessory/bracelet/material/platinum
+	bracelettype["bracelet, glass"] = /obj/item/clothing/accessory/bracelet/material/glass
+	bracelettype["bracelet, wood"] = /obj/item/clothing/accessory/bracelet/material/wood
+	bracelettype["bracelet, plastic"] = /obj/item/clothing/accessory/bracelet/material/plastic
+	gear_tweaks += new/datum/gear_tweak/path(bracelettype)
+
+/datum/gear/accessory/bracelet/friendship
+	display_name = "friendship bracelet"
+	path = /obj/item/clothing/accessory/bracelet/friendship
+	cost = 0
```
Comment (Anewbe) [inline code/modules/client/preference_setup/loadout/loadout_accessories.dm]: Why is this cost = 0 ?

Comment (elgeonmb) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm]: friendship is priceless

**Comment by Belsima** [comment]
what are the OBJ sprites like ?

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm (hunk 1 above)
Comment (mistyLuminescence) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm]: Ten points to @elgeonmb - you can't put a price on friendship!

**Comment by mistyLuminescence** [comment]
@Belsima [Friendship](https://image.prntscr.com/image/G10uLzJgR3CYC-PsbRl-Tw.png) and [regular/material](https://image.prntscr.com/image/J-XzUcPTSLeFKCHKHgdffQ.png). Material bracelet changes color when it's a different material (e.g. wooden ones are brown).

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm (hunk 1 above)
Comment (Anewbe) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm]: Make it consistent, please.

**Pull request merged**